recipes/brotli-asgi/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "brotli-asgi" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: bcb815ba0c7b49a78df77bdede0d3fa62daaa429a326a1dcac293d69ef94858f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - brotli-python >=1.0.7
+    - pip
+    - python
+    - starlette >=0.13.4
+  run:
+    - brotli-python >=1.0.7
+    - python
+    - starlette >=0.13.4
+
+test:
+  imports:
+    - brotli_asgi
+
+about:
+  home: "https://github.com/fullonic/brotli-asgi"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.md
+  summary: "A compression AGSI middleware using brotli"
+  doc_url: 
+  dev_url: 
```

Comment:
Don't leave empty keys. You can remove those if you don't want to add the optional metadata there.

Reply:
done